### ISSUE TITLE
Mod is not adding anything into the game.

### ISSUE THREAD

**Issue by bajungadustin** [ManlyMarco/ABMX#18]
I added the mod to the game exactly as instructed. 
Current version of BepInEx 5.0.1.0
Illusion Mod API v1.11.1
and ABMX v4.3

and even checked with KK manager which states that both plugins are enabled but I get no additional sliders in game.

**Comment by ManlyMarco** [comment]
Send your output_log file from game directory.

**Comment by Valenilluminati** [comment]
I have the same problem, here it is the output log
[output_log.txt](https://github.com/ManlyMarco/ABMX/files/4778740/output_log.txt)

**Comment by DerekEvans7** [comment]
i downloaded someone elses mod and it worked fine. not sure why this one didnt but i no longer have this mod or the log

**Comment by ManlyMarco** [comment]
It sounds like an issue with your setup, either install latest HF Patch for your game or ask for help on the appropriate discord server so we can help you there.

**Comment by DerekEvans7** [comment]
I dont need help anymore. I downloaded a different patch that was a collection of a bunch of patches including this one. Installing that mod group using their method worked to successfully install this mod while following the instructions for installing this mod did not. I followed both sets of instructions to the letter. So I seems like the issue might be that the instructions here were maybe not as clear as the instructions for the other grouped mods as this mod was able to be loaded without issue using their method.

Either way I no longer need help.
